tests/test_integrated_channels/test_degreed/test_exporters/test_course_metadata.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for Degreed course metadata exporters.
+"""
+
+from __future__ import absolute_import, unicode_literals, with_statement
+
+import json
+import unittest
+
+import mock
+import responses
+from integrated_channels.degreed.exporters.course_metadata import DegreedCourseExporter
+from pytest import mark
+
+from test_utils import factories
+from test_utils.fake_enterprise_api import EnterpriseMockMixin
+
+
+@mark.django_db
+class TestDegreedCourseExporter(unittest.TestCase, EnterpriseMockMixin):
+    """
+    Tests for the ``DegreedCourseExporter`` class.
+    """
+
+    def setUp(self):
+        self.enterprise_customer = factories.EnterpriseCustomerFactory()
+        self.mock_ent_courses_api_with_pagination(
+            enterprise_uuid=str(self.enterprise_customer.uuid),
+            course_run_ids=['course-v1:edX+DemoX+Demo_Course_1']
+        )
+        self.config = factories.DegreedEnterpriseCustomerConfigurationFactory(
+            enterprise_customer=self.enterprise_customer,
+            degreed_company_id='orgCode'
+        )
+
+        jwt_builder = mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
+        self.jwt_builder = jwt_builder.start()
+        self.addCleanup(jwt_builder.stop)
+        super(TestDegreedCourseExporter, self).setUp()
+
+    @responses.activate
+    def test_export(self):
+        """
+        If a course is unenrollable, it should be returned with a JSON dump and the ``DELETE`` method.
+        """
+        exporter = DegreedCourseExporter('fake-user', self.config)
+        # Add a course marked for deletion by giving it just the single key; the course run ID.
+        exporter.courses.append({'contentId': 'course-id-for-deletion'})
+        data = exporter.export()
+        json_dump1, method1 = (
```

Comment:
@UmanShahzad It would be nice to be a bit more specific here (and in L62 below). How about something like `json_dump_deleted, delete_method` (and `json_dump_posted, post_method`)?

In any case, you could update the inline comments ("JSON Dump 1", "Method 1", etc.) to be a bit more specific.

Reply:
:+1: